corehq/form_processor/models.py

```diff
@@ -1216,6 +1216,14 @@ class LedgerValue(DisabledDbMixin, models.Model, TrackRelatedChanges):
     last_modified = models.DateTimeField(auto_now=True)
 
     @property
+    def last_modified_date(self):
+        return self.last_modified
+
+    @property
+    def product_id(self):
+        return self.entry_id
```

Comment:
Yea, this is basically just for compatibility with the `StockState` class so that they can be used interchangeably during restore (and other places) without additional boilerplate to check which property name is needed.